src/scores/continuous.py

```diff
@@ -39,19 +41,38 @@ def mse(fcst, obs, reduce_dims=None, preserve_dims=None, weights=None):
             Otherwise: Returns an object representing the mean squared error,
             reduced along the relevant dimensions and weighted appropriately.
     """
+    as_pandas_series = False
+    both_pandas = False
+    if type(fcst) == pandas.Series:
+        fcst = fcst.to_xarray()
+        as_pandas_series = True
+
+    if type(obs) == pandas.Series:
+        obs = obs.to_xarray()
+        as_pandas_series = True
+        if as_pandas_series == True:
+            both_pandas = True
+
```

Comment:
While that's true, I think it would make the implementation more complex. I'll keep it in mind for the future, particularly if the same approach is taken in any additional functions.